src/compiler/types.ts

```diff
@@ -8693,6 +8708,7 @@ namespace ts {
         readonly includePackageJsonAutoImports?: "auto" | "on" | "off";
         readonly provideRefactorNotApplicableReason?: boolean;
         readonly jsxAttributeCompletionStyle?: "auto" | "braces" | "none";
+        readonly overrideImportMode?: false | "require" | "import";

```

Comment:
I don’t think this makes sense as a “user preference”—seems like it should just be another parameter to `getModuleSpecifiers`. These all have user-facing analogs in protocol.ts and settings in VS Code. It’s kind of a coincidence that a couple of them are useful for declaration emit.